vnext/Microsoft.ReactNative/ABIModule.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ABIModule.h"
+#include "NativeModulesProvider.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+auto ABIModule::getName() -> std::string {
+  auto name = m_module.Name();
+  return to_string(name);
+}
+
+auto ABIModule::getConstants() -> std::map<std::string, folly::dynamic> {
+  std::map<std::string, folly::dynamic> constants;
+  auto constantsMapView = m_module.Constants();
+
+  for (auto const &pair : constantsMapView) {
+    std::string key = to_string(pair.Key());
+    folly::dynamic value = ConvertToDynamic(pair.Value());
+
+    constants.emplace(key, value);
```

Comment:
`constants.emplace(std::move(key), std::move(value));`

Reply:
Including this change as part of my upcoming "samples" PR.